hera_mc/tests/__init__.py

```diff
@@ -18,10 +18,15 @@
 
 
 def redis_online():
+    # redishost = 'localhost'
+    redishost = None
```

Comment:
This addition seems very weird to me, but I'd like @mkolopanis's input.

Reply:
This and the additions of the lines below looks like additions made to make redis testing work locally with a redis server not hosted at "redishost". I'm fine if you need to bend over a little bit to get testing working a local install, but I would prefer not to have it in the actual codebase online.

Reply:
I fixed this so that there is a TEST_DEFAULT_REDIS_HOST which is handled consistently throughout tests.

Reply:
Fixed this via zoom meeting